src/components/e-fullwidth.vue

```diff
@@ -71,29 +79,31 @@
       }
     },
     mounted() {
-      if (!this.resizeObserver) { // IE11 fallback.
+      if (!this.resizeObserver) {
         this.updateScrollbarWidth();
       }
     },
     // beforeUpdate() {},
     // updated() {},
     // activated() {},
     // deactivated() {},
-    beforeDestroy() {
+    beforeUnmount() {
       if (this.resizeObserver) {
         this.resizeObserver.unobserve(window.document.body);
-      } else { // IE11 fallback.
+      } else {
         window.removeEventListener('resizeend', this.updateScrollbarWidth);
       }
     },
-    // destroyed() {},
+    // unmounted() {},
 
     methods: {
       /**
        * Recalculates the scrollbar width.
        */
       updateScrollbarWidth() {
-        clearTimeout(this.resizeTimeout);
+        if (this.resizeTimeout) {
```

Comment:
This should not be required. `clearTimeout` normally can handle existing and missing sessions.